### ISSUE TITLE
As a community we need a Code of Conduct

### ISSUE THREAD

**Issue by mjy** [tdwg/developers#2]
How do people feel about using on of the default options provided by Github?

**Comment by niconoe** [comment]
Yeah, we need one and I'd use something simple and standard to start. We can always change it later if there's an issue, there are good chances that it'll never happen.

So if the GitHub suggestions look good to you (I briefly checked Contributor Covenant, seems ok for me!), please go ahead.

Just my 2 cents!

**Comment by damianooldoni** [comment]
I find the code of conduct of Data Carpentry very inspiring:
https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
Obviously it has to be slightly rearranged as everything happens online in our case.
If you all find it worth, I can do it and start a PR.
